NineChronicles.Headless/GraphTypes/Input/ClaimDataInputType.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL.Types;
+using Libplanet.Crypto;
+using Libplanet.Explorer.GraphTypes;
+using Libplanet.Types.Assets;
+
+namespace NineChronicles.Headless.GraphTypes.Input;
+
+public class ClaimDataInputType : InputObjectGraphType<(Address avatarAddress, FungibleAssetValue fav)>
```

Comment:
Maybe it should type `IReadOnlyList<FungibleAssetValue> favs` or implement `InputObjectGraphType` (non-generic) instead. I'm not sure why the second element type is just `FungibleAssetValue` 👀